docs/remote.rst

```diff
@@ -10,7 +10,7 @@ tests that you can successfully run locally. And you
 have a ssh-reachable machine ``myhost``.  Then
 you can ad-hoc distribute your tests by typing::
 
-    pytest -d --tx ssh=myhostpopen --rsyncdir mypkg mypkg
+    pytest -d --tx ssh=myhostpopen --rsyncdir mypkg
```

Comment:
```suggestion
    pytest -d  --rsyncdir mypkg --tx ssh=myhostpopen mypkg/tests/unit/test_sometihng.py
```

might drive it home better, the rsyncdir is indeed a rather confusing option, personally i never used it

Reply:
Great, this makes more sense.